README.md

```diff
@@ -1,28 +1,47 @@
 # Travelers App - Web and API
 
-### Will make a seperate repo for mobile soon after we make some headway here and decide on tech for the native app
+## Locations
 
-- Install docker(&& docker-compose) if you haven't already
 - client will run on http://localhost:3000
 - GraphQL API test page is at http://localhost:5000/graphql very cool tool, link in docker output to terminal also
 
-To get started, run the following command in the root of the project:
+---
+
+### Set-up
 
+To get started, run one of the following commands in the root of the project:
+
+```
+npm install
+yarn install
```

Comment:
We should probably come to a consensus on which we want to use, or the lock files can become out of sync.  Is that why they were added to `.gitignore`?

Reply:
Yeah i had left them out so people could choose the package manger they prefer but i see now how that could complicate things